kakarot_scripts/deployment/dualvm_token_deployments.py

```diff
@@ -0,0 +1,188 @@
+# %% Imports
+import json
+import logging
+from typing import Any, Dict, List
+
+from uvloop import run
+
+from kakarot_scripts.constants import (
+    EVM_ADDRESS,
+    NETWORK,
+    RPC_CLIENT,
+    TOKEN_ADDRESSES_DIR,
+    NetworkType,
+)
+from kakarot_scripts.utils.kakarot import deploy as deploy_kakarot
+from kakarot_scripts.utils.kakarot import deploy_and_fund_evm_address
+from kakarot_scripts.utils.kakarot import dump_deployments as dump_evm_deployments
+from kakarot_scripts.utils.kakarot import get_deployments as get_evm_deployments
+from kakarot_scripts.utils.starknet import deploy as deploy_starknet
+from kakarot_scripts.utils.starknet import execute_calls, get_class_hash_at
+from kakarot_scripts.utils.starknet import get_contract as get_contract_starknet
+from kakarot_scripts.utils.starknet import get_deployments as get_starknet_deployments
+from kakarot_scripts.utils.starknet import (
+    get_starknet_account,
+    invoke,
+    register_lazy_account,
+    remove_lazy_account,
+)
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+# %%
+
+
+async def deploy_starknet_token(owner_address: str) -> Any:
+    address = await deploy_starknet("StarknetToken", int(1e18), owner_address)
```

Comment:
decimals should come from the token list

Reply:
> decimals should come from the token list

There's no notion of decimals in the tokenlist. Plus, we're only deploying these tokens for testing purposes on dev networks. Thus I don't consider keeping 18 default decimals problematic